src/ontology/mondo-edit.obo

```diff
@@ -417628,6 +417724,18 @@ is_a: MONDO:0004298 {source="DOID:0040090", source="NCIT:C95752/inferred"} ! sto
 intersection_of: MONDO:0004966 ! gastritis (disease)
 intersection_of: disease_arises_from_feature HP:0002960 ! Autoimmunity
 
+[Term]
+id: MONDO:0031178
```

Comment:
only has one child, seems suspect...

Reply:
there is another child, that has yet be brought into Mondo, Nico said it will come in the next slurp:
https://www.omim.org/entry/619041
https://github.com/monarch-initiative/mondo/issues/2539#issuecomment-767176000